logger.go

```diff
@@ -121,9 +125,15 @@ func (sumoLogger *sumoLogger) handleBatchedLogs() {
     for {
       err := sumoLogger.sendLogs(logBatch)
       if err == nil {
+        retryInterval = initialRetryInterval
         break
       }
+      logrus.Info(fmt.Sprintf("Sleeping for %s before retry...", retryInterval.String()))
```

Comment:
`retryInterval.String()` converts the `retryInterval` from `time.Duration` type to string with units attached like `2s` or `500ms`